amqp/channelpool/pool.go

```diff
@@ -154,6 +163,14 @@ func (p *pool) Get(ctx context.Context) (*amqp.Channel, error) {
 	}
```

Comment:
should we here put the get in a for loop and keep on trying to pull out a channel as long as the context is not done?

Reply:
Yep, I dunno what was expected by the caller, but that's definitely an option. Will do

Reply:
Any risk to flood the rabbitMQ cluster with channels doing this ? 

If one host nodes goes down, all the node that attempt to consume one of the queues of this host node will spam the other rabbitMQ nodes. 

I would go for a backoff in that case, wdyt ?